usecases/modulecomponents/usage/base_module.go

```diff
@@ -231,7 +238,8 @@ func (b *BaseModule) collectAndUploadUsage(ctx context.Context) error {
 	b.mu.Lock()
 	defer b.mu.Unlock()
 	usage.Version = b.policyVersion
-	usage.CollectingTIme = collectionTime
+	usage.CollectingTime = collectionTime
+	usage.Schema = b.schemaGetter.GetSchemaSkipAuth().Objects
```

Comment:
comment for the future: we should change at some point this variable name `Objects` to `Schema` it's misleading now. This is a leftover from migration to `v1`, I forgot to refactor it :)